main-command/src/main/scala/sbt/Command.scala

```diff
@@ -16,6 +16,7 @@ import sbt.internal.util.{ AttributeKey, AttributeMap, Util }
  * Typically you would resort to a command when you need to do something that's impossible in a regular task.
  */
 sealed trait Command {
+  def name: Option[String]
```

Comment:
Let's add this with an extension method defined in the `Command` object. That way we don't have to change SimpleCommand.

Reply:
I think this is indeed a superior way to do it 👍